docs/user/openstack/install_upi.md

```diff
@@ -336,22 +353,29 @@ values:
 
 <!--- e2e-openstack-upi: INCLUDE START --->
 ```sh
-$ python -c 'import yaml
+$ python -c 'import os
+import yaml
+os_net_id = os.getenv("OS_NET_ID")
+path = "common.yaml"
+facts = yaml.safe_load(open(path))[0]["tasks"][0]["set_fact"]
+os_network = facts["os_network"].replace("{{ os_net_id }}", os_net_id)
+os_subnet = facts["os_subnet"].replace("{{ os_net_id }}", os_net_id)
+os_subnet6 = facts["os_subnet6"].replace("{{ os_net_id }}", os_net_id)
 path = "install-config.yaml"
 data = yaml.safe_load(open(path))
 inventory = yaml.safe_load(open("inventory.yaml"))["all"]["hosts"]["localhost"]
 machine_net = [{"cidr": inventory["os_subnet_range"]}]
 api_vips = [inventory["os_apiVIP"]]
 ingress_vips = [inventory["os_ingressVIP"]]
-ctrl_plane_port = {"network": {"name": inventory["os_network"]}, "fixedIPs": [{"subnet": {"name": inventory["os_subnet"]}}]}
-if inventory.get("os_subnet6"):
+ctrl_plane_port = {"network": {"name": os_network}, "fixedIPs": [{"subnet": {"name": os_subnet}}]}
+if inventory.get("dualstack"):
```

Comment:
This condition needs to be updated to use `os_subnet6_range` instead

Reply:
ahh, right.